### DISCUSSION TITLE
why can't I create widget in local development?

### DISCUSSION THREAD

**Discussion by YahyaDalbah** [chatwoot/chatwoot#10355] [Chatwoot APIs]
![Capture](https://github.com/user-attachments/assets/d284d821-50e5-4c1c-84fd-da2b54755f89)
when i go to http://localhost:3000/widget_tests
or when i use the script provided by chatwoot for creating a website widget, it doesn't show and always return a 404 error and http://localhost:3000/packs/js/sdk.js doesn't exist

**Comment by sojan-official** [comment]
please share the steps you have taken to do local development

**Reply by YahyaDalbah** [reply]
the exact steps provided by the contribution guide
except i ran make `db_create` , `make db_migrate` , `make db_seed` instead of `make db`
I'm using ubuntu 24.04.1 LTS

**Reply by LevBaranov** [reply]
So, three months later.. I think you need to run the script in package.json: build:sdk. For example: `pnpm run build:sdk`
